### PR TITLE
refactor: rename default config as "main"

### PR DESCRIPTION
**Pull request by snyamathi**
fixes: https://github.com/yahoo/ycb/issues/78

This PR allows the default config to be `"main"` while retaining the ability to use the previous keyword, `"master"`.

There is a private instance variable, `this.masterDelta` which is not part of the public API, but was accessible previously. I don't think this needs to be considered breaking, since none of the public API is changed.

I also found one small bug where we share a reference to the lookup table between YCB instances. I don't know how this wasn't an issue before, but I've fixed it in this PR as well. Basically `this.numberToValue` for all YCB instances would point to the shared object reference to `DEFAULT_LOOKUP`.

cc @BillDorn @redonkulus 

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -595,22 +594,22 @@ Ycb.prototype = {
                 }
                 intervals[i] = interval;
             }
-            if (settingDimensions[0] === 'master') {
+            if (settingDimensions[0] === 'main' || settingDimensions[0] === 'master') {
```
Comment (snyamathi) [inline index.js:597]: This is really the entirety of the change in this one line.

**Review by redonkulus — APPROVED** [review]
LGTM

**Comment by dsaint** [comment]
Will ycb-config get a similar cleanup?

**Comment by snyamathi** [comment]
We should do that repo as well, yes. Though it's only tests/docs in that case with no actual code references

**Pull request merged**